nixos/modules/services/databases/postgresql.nix

```diff
@@ -165,25 +165,13 @@ in
               '';
             };
 
-            ensurePermissions = mkOption {
```

Comment:
Could we deprecate this, instead of outright deleting the option?

Reply:
There will be a mkOptionRemoved when this is out of draft

Reply:
I don't like the idea of removing the option outright. There are valid use cases for `ensurePermissions`, that are not affected by the pg15 change. Maybe we can put a warning about this, when the option is used?